File: CPE-1star/UVA272-TeX Quotes .py
#第一個"換成``，第二個換成''，以此下去改
count=0
while True:
	try:
		text=input()
		for i in range(len(text)):
			if text[i]=='"':
				if count%2==0:
					print("``",end="")
				else:
					print("''",end="")
				count+=1
			else:
				print(text	[i],end="") 
		print()
	except EOFError:
		break

File: CPE-1star/UVA272-TeX Quotes.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    string text;
    vector<string> strs;
    int count = 0;
    while (getline(cin, text)) {
        strs.push_back(text);
    }
    for (int i = 0; i < strs.size(); i++) {
        string line = strs[i];
        for (int j = 0; j < line.size(); j++) {
            if (line[j] == '"') {
                if (count % 2 == 0)
                    cout << "``";
                else
                    cout << "''";
                count += 1;
            }
            else
                cout << line[j];
        }
        cout << endl;
    }
    return 0;
}
